src/components/notes/EditorCorner.spec.jsx

```diff
@@ -10,13 +13,28 @@ import EditorCorner from './EditorCorner'
 describe('EditorCorner', () => {
   const mockDoc = {}
   it('shows the sharing widget on private views', () => {
-    const readOnlyComponent = shallow(
-      <EditorCorner doc={mockDoc} isPublic={false} isReadOnly={true} />
+    //  useI18n.mockReturnValue({})
+    const readOnlyComponent = mount(
+      <EditorCorner doc={mockDoc} isPublic={false} isReadOnly={true} />,
+      {
+        wrappingComponent: I18n,
+        wrappingComponentProps: {
+          lang: 'en',
+          dictRequire: () => en
+        }
+      }
```

Comment:
Nit: It is useful to make an AppLike component containing all the wrappers necessary for tests.

Reply:
Thanks!